aleph/logic/collections.py

```diff
@@ -102,7 +102,7 @@ def delete_collection(collection, keep_metadata=False, sync=False):
     deleted_at = collection.deleted_at or datetime.utcnow()
     Entity.delete_by_collection(collection.id, deleted_at=deleted_at)
     Document.delete_by_collection(collection.id)
-    Mapping.delete_by_collection(collection.id)
+    Diagram.delete_by_collection(collection.id)
```

Comment:
Did you mean to get rid of the mapping delete?

Reply:
Nope, mistake while merging I guess